src/nativeMain/kotlin/com/autodesk/coroutineworker/CoroutineWorker.kt

```diff
@@ -97,9 +97,7 @@ public actual class CoroutineWorker internal actual constructor() {
                         repeatedlyCheckForCancellation(this.coroutineContext, cancelled) { completed }
                         // inside of a new CoroutineScope, so that child jobs are cancelled
                         coroutineScope {
-                            autoreleasepool {
```

Comment:
@benasher44 , any thoughts on this?

If we use the autorelease pool we end up with a test failure on `withContext_across_threads`.

```
objc[14264]: Invalid or prematurely-freed autorelease pool 0x7fd7dc80c048.
Child process terminated with signal 6: Abort trap
```

I'm uncertain why it is either invalid or prematurely freed, but I also don't know that we need the release pool.

Reply:
@DanielRBaird actually can you make a test that reproduces it, possibly? I'm surprised all tests have passed.

Reply:
Yeah, they pass with the autorelease pool removed like I did here. If you undo this change it will fail.

Reply:
Ah! Can you put it back? I'd like to open a KT ticket with the failure. This is prob a K/N bug

Reply:
Okay according to [the ticket](https://youtrack.jetbrains.com/issue/KT-50712), we can just remove this. This was added in #56, which predates Kotlin 1.4 (autoreleasepools added to Worker in 1.4.30)

Reply:
Yep. I'll do that. Thanks for helping investigate!